feature/src/main/java/com/gdg/feature/calendar/CalendarRoute.kt

```diff
@@ -1,51 +1,394 @@
+
+                Text(
+                    text = stringResource(R.string.calender_sub_header_1),
+                    style = CrowdZeroTheme.typography.h3Regular,
+                    color = CrowdZeroTheme.colors.gray700
+                )
+                Text(
+                    text = stringResource(R.string.calender_sub_header_2),
+                    style = CrowdZeroTheme.typography.h3Regular,
+                    color = CrowdZeroTheme.colors.gray700
+                )
+            }
+
+            Image(
+                painter = painterResource(R.drawable.ic_alert),
+                contentDescription = null,
+                modifier = Modifier.size(105.dp)
+            )
+        }
+
+        // 캘린더 컴포넌트
+        CalendarComponent(
+            currentMonth = currentMonth,
+            selectedDate = selectedDate,
+            onMonthChange = { currentMonth = it },
+            onDateSelected = onDateSelected
         )
+
+        Spacer(
+            modifier = Modifier
+                .height(1.dp)
+                .fillMaxWidth()
+                .background(CrowdZeroTheme.colors.gray500)
+        )
+
+        Spacer(modifier = Modifier.height(25.dp))
+
+        // 선택한 날짜 표시
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Text(
+                text = selectedDate.format(DateTimeFormatter.ofPattern("yyyy년 MM월 dd일")),
+                style = CrowdZeroTheme.typography.h3Bold,
+                color = CrowdZeroTheme.colors.gray800
+            )
+            Spacer(modifier = Modifier.width(8.dp))
+            Text(
+                text = selectedDate.dayOfWeek.getDisplayName(java.time.format.TextStyle.FULL, Locale.KOREAN), // 한글 요일 표시
+                style = CrowdZeroTheme.typography.h3Bold,
+                color = CrowdZeroTheme.colors.green700
+            )
+        }
+
+        Spacer(modifier = Modifier.height(18.dp))
+
+        if (scheduleList.isEmpty()) {
+            // 정보 없는 경우 메시지 표시
+            Spacer(modifier = Modifier.height(80.dp))
+            Text(
+                text = "해당 날짜의 집회 정보가 없습니다",
+                style = CrowdZeroTheme.typography.h5Medium,
+                color = CrowdZeroTheme.colors.gray900,
+                modifier = Modifier.padding(horizontal = 16.dp)
+            )
+        } else {
+            LazyColumn(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(horizontal = 16.dp),
+                verticalArrangement = Arrangement.spacedBy(10.dp)
+            ) {
+                items(scheduleList) { schedule ->
+                    CalendarInfoBox(data = schedule)
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun CalendarInfoBox(data: ScheduleEntity) {
+    Box(
+        modifier = Modifier
+            .size(370.dp, 85.dp)
```

Comment:
P1: 이렇게 사이즈를 지정해줄 경우 폰 기종에 따라 크기가 원하는 비율대로 안나오게 됩니다. 크기 지정은 최대한 지양해주시고 fillMaxWidth와 padding으로 크기를 조정해야 합니다.
```suggestion
            .fillMaxWidth()
```